PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -1519,6 +1519,106 @@ HFONT createFont(const TCHAR* fontName, int fontSize, bool isBold, HWND hDestPar
 	return newFont;
 }
 
+bool isRawWin32FileName(const generic_string& fileName)

```

Comment:
And please add the following comment above the function definition:
```cpp
// "For file I/O, the "\\?\" prefix to a path string tells the Windows APIs to disable all string parsing and to send the string that follows it straight to the file system..."
// ref: https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#win32-file-namespaces
```

Reply:
I have started to use the "raw" here because of I saw it somewhere and I did not have a better short name for it.

The MS long term for it is "The Win32 namespace prefixing", so it should rather be "isWin32NamespacePrefixedFileName", ok?

Reply:
@xomx 
Yep! OK for "isWin32NamespacePrefixedFileName".